framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -294,14 +297,17 @@ public void parse(Map<Element, AnnotatedTypeMirror> atypes, Map<String, Set<Anno
         parse(this.index, atypes, declAnnos);
     }
 
-    private void parse(IndexUnit index, Map<Element, AnnotatedTypeMirror> atypes, Map<String, Set<AnnotationMirror>> declAnnos) {
```

Comment:
There were a few very long lines in this file and I tried to break all of them down to something more manageable.